src/app/api/v1/share-links/[id]/qrcode/route.ts

```diff
@@ -45,18 +38,20 @@ const shlinkRepo = container.get<ISHLinkRepository>(SHLinkRepositoryToken);
  *               format: binary
  */
 
-export async function POST(request: Request, { params }: {params: { id: string } }) {
-    
+export async function POST(
+  request: Request,
+  { params }: { params: { id: string } },
+) {
   try {
     const requestDto: SHLinkQRCodeRequestDto = await request.json();
     const { id } = params;
 
     let shlink = await getSingleSHLinkUseCase(
-      {repo: shlinkRepo},
-      {id: id, managementToken: requestDto.managementToken},
+      { repo: shlinkRepo },
+      { id: id, managementToken: requestDto.managementToken },
```

Comment:
Hello @jacob-khoza-symb, 
A tiny thing here maybe if you want to simplify this
(based on the suggestion of the previous Comment)

```suggestion
      { id,managementToken },
```